src/plugins/flare/flareplugin.cpp

```diff
@@ -361,10 +432,16 @@ bool FlarePlugin::write(const Tiled::Map *map, const QString &fileName)
                     out << "," << w << "," << h << "\n";
 
                     // write all properties for this object
-                    Properties::const_iterator it = o->properties().constBegin();
-                    Properties::const_iterator it_end = o->properties().constEnd();
-                    for (; it != it_end; ++it) {
-                        out << it.key() << "=" << it.value().toString() << "\n";
+                    QVariantMap propsMap = o->properties();
```

Comment:
If you put a `const` in front of it you don't need to bother with explicitly calling `constBegin`/`constEnd`.